distribution/ddf-common/src/main/resources/etc/system.properties

```diff
@@ -137,6 +137,16 @@ org.codice.allowBasicAuthOverHttp=false
 
 # Sets the default port number for the catalog-ftp feature FTP endpoint
 org.codice.ddf.catalog.ftp.port=8021
+#
+# Maximum Endpoint Contact Interval
+#
+# Max seconds between attempts to contact the endpoint of a source
+org.codice.ddf.platform.util.http.maxRetryInterval=300
```

Comment:
❗️  Based on your algorithm update, to double the initial retry value up to by not exceeding this max value, it would probably make more sense to set the default here to either 240 or 480 seconds (probably the former). We're never going to get to 300, even though I believe it sets an expectation we will based on this max value.